src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -33,7 +34,14 @@ public class ServerJob : IIdentifiable
 
         [JsonConverter(typeof(StringEnumConverter))]
         public Scheme Scheme { get; set; }
-
+        public string AspNetCoreVersion { get; set; }
+        public string RuntimeFrameworkVersion { get; set; }
+        public string Session { get; set; }
+        public string Description { get; set; }
+        public TimeSpan Startup { get; set; }
+        public TimeSpan Latency { get; set; }
+        public List<long> WorkingSets { get; set; } = new List<long>();
+        public List<double> Cpus { get; set; } = new List<double>();
```

Comment:
I think it would be slightly better to associate the elapsed time with each `WorkingSet` and `CPU` data point, so it would be possible to say create a graph of them over time.  Even if we don't store this in SQL today.  So I would change these to something like `List<Tuple<TimeSpan, long>>` or create a custom type.

Reply:
Was too lazy to create a dedicated class (better than a tuple for JSon), but will do then.